### PR TITLE
Fix @liveblocks/react bundle

### PR DESCRIPTION
**Pull request by marcbouchenoire**
This PR adds `nanoid` as a dependency as it should have been.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -19187,6 +19187,7 @@
       "dependencies": {
         "@liveblocks/client": "1.3.0",
         "@liveblocks/core": "1.3.0",
+        "nanoid": "^4.0.2",
```
Comment (nvie) [inline package-lock.json:19190]: Thinking about this… I think maybe we need to make this a `peerDependencies` instead, just to facilitate people that already may be using (a different version of) `nanoid` in their frontend projects?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# v1.3.1
+
+### `@liveblocks/react`
+
+- Fix bug that would cause Node.js-specific imports to be present in the final
+  bundle
```
Comment (nvie) [inline CHANGELOG.md:6]: Maybe just say

```suggestion
- Fix missing dependency declaration
```

**Review by nvie — APPROVED** [review]
Approved… but with one consideration.

File: package-lock.json (hunk 1 above)
Comment (marcbouchenoire) [reply package-lock.json:19190]: Do `peerDependencies` auto-install nowadays?

Comment (nvie) [reply package-lock.json:19190]: They do these days!

Screen Shot 2023-09-08 at 11 16 10@2x

Comment (nvie) [reply package-lock.json:19190]: (And that makes a lot of sense.)

File: packages/liveblocks-react/package.json [hunk 3]
```diff
@@ -35,10 +35,10 @@
   "dependencies": {
     "@liveblocks/client": "1.3.0",
     "@liveblocks/core": "1.3.0",
-    "nanoid": "^4.0.2",
     "use-sync-external-store": "^1.2.0"
   },
   "peerDependencies": {
+    "nanoid": "^3 || ^4",
```
Comment (nvie) [inline packages/liveblocks-react/package.json:41]: Oh, that's clever. That lets people decide which one to use! I like it 👍

Comment (marcbouchenoire) [reply packages/liveblocks-react/package.json:41]: The issue I have with that is that if `^4` is picked (e.g. automatically), our CJS bundle might be incorrect depending on the final bundler or lack of it. I think the ideal setup is to add it as a normal dependency as `"nanoid": "^3.3.6"`.

**Review by nvie — APPROVED** [review]
Yesss, this looks great now! 👍

**Pull request merged**